homework_4/bet_acceptance_api/internal/infrastructure/rabbitmq/betPublisher.go

```diff
@@ -0,0 +1,83 @@
+package rabbitmq
+
+import (
+	"encoding/json"
+	uuid "github.com/nu7hatch/gouuid"
+	"github.com/pkg/errors"
+	"github.com/streadway/amqp"
+	models2 "github.com/superbet-group/code-cadets-2021/homework_4/bet_accceptance_api/internal/api/controllers/models"
```

Comment:
same here, you are leaking controller logic to the publisher.
Publisher should only know for rabbit models